### ISSUE TITLE
count_ss_match uses wrong file

### ISSUE THREAD

**Issue by johnlamb** [multicom-toolbox/CONFOLD#1]
Hi
See https://github.com/multicom-toolbox/CONFOLD/blob/6b29b50814a654992adb908342a3e0ca5c6a5a01/confold.pl#L1459

In the count_ss_match function, $file_sec is sent in as a parameter(in both instances when the function is used it is the secondary structure file that is sent in). However, this file is never referenced and %residue_ss1 is instead created from $file_fasta. 
This causes the later comparison to compare amino acid vs secondary structure characters which will only work in cases of H, E and C and will hence give a wrong secondary structure match count.
I am happy to do a pull request with a correction if you can confirm it is a bug.

**Comment by badriadhikari** [comment]
Yes, I confirm that this is a bug and it should be fixed. This is extremely trivial though. The comparison results are only used for visualization/logging purposes and do not impact any model generation and selection.

**Comment by jianlin-cheng** [comment]
Would you like for the user to fix it or to fix it by yourself? You can
reply to the user with your plan.

Thanks.

Jianlin

**Comment by badriadhikari** [comment]
This is taken care!

**Comment by jianlin-cheng** [comment]
Great. Thanks! --Jianlin

**Comment by TaufiqIslamProtick** [comment]
Dear Professor Adhikari

The README part section 5.2 runs a script with parameters that takes input a sequence, contacts and secondary structure data and other related parameters to run the program. Everything up to this point runs without any errors. When this line is run in the terminal, 1.error.png (bold yellow box) happens. When I checked the output folder I saw in 2.extended.mtf.png(bold green box) that the extended.mtf file is created. (Although the timestamp in the two images are different, extended.mtf is created every time I run the shell script written at README Section 5.2). 

Did I miss something, or it needs a special fix to run it properly in my local computer? It would be so kind of you if you shared your insights. I am attaching the snapshots for your convenience. 

![1error](https://user-images.githubusercontent.com/25647811/53402504-10442300-39dc-11e9-9c6b-b894fcf589c0.png)
![2extended mtf](https://user-images.githubusercontent.com/25647811/53402512-15a16d80-39dc-11e9-9e6a-90c54f1e7cb4.png)
